pkg/cleaner/cleaner.go

```diff
@@ -140,13 +143,48 @@ func (c *CleanupStatusTracker) CancelJob(bdName string) error {
 
 // runJob creates a new cleanup job in the namespace
 func (c *Cleaner) runJob(bd *v1alpha1.BlockDevice, volumeMode VolumeMode) error {
-	job, err := NewCleanupJob(bd, volumeMode, c.Namespace)
+
+	//retreive node Object to pass tolerations to the Job
+	selectedNode, err := c.getNodeObjectByNodeName(bd.Labels[controller.KubernetesHostNameLabel])
```

Comment:
If this is the only place that you are using `controller` to get the constant, then it is better to directly use the constant then import it. Also should this be HostName or nodeName?

Reply:
cc: @akhilerm

Reply:
This will give the hostname @kmova  and @rahulchheda . The method should be `getNodeObjectByHostName`. I think in localpv also we are using the same method. We can filter from nodes based on the hostname label.